app/assets/javascripts/components/Dashboard.jsx

```diff
@@ -1,18 +1,35 @@
 class Dashboard extends React.Component {
   render() {
-  	const { Tabs, Tab } = ReactBootstrap;
+  	const { Tab, Row, Col, Nav, NavItem } = ReactBootstrap;
   	const { user } = this.props;
+  	const fullName = `${user.first_name} ${user.last_name}`
     return (
     	<div>
-	    	<DashboardHeader name={user.first_name}/>
-	    	<Tabs defaultActiveKey={1} id="user-dashboard">
-	    		<Tab eventKey={1} title="Tasks">
-	    			<TaskList userId={user.id} />
-	    		</Tab>
-	    		<Tab eventKey={2} title="Clients">
-	    			<ClientList />
-	    		</Tab>
-	    	</Tabs>
+	    	<DashboardHeader name={fullName}/>
+	    	<Tab.Container id="dashboard-nav" defaultActiveKey={1}>
+			    <Row className="clearfix">
```

Comment:
weird tab stuff here lol, otherwise LGTM

Reply:
i have no idea why my tab stuff is always messed up